src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes } from '../../utils/propUtils'
+import Icon from 'src/elements/Icon/Icon'
+
+/**
+ * A divider sub-component for Breadcrumb component.
+ */
+function BreadcrumbDivider(props) {
+  const {
+    children, icon, className, ...rest,
+  } = props
+
+  const classes = cx(
+    className,
+    'divider',
+  )
+
+  if (icon) {
+    return <Icon className={classes} name={icon} {...rest} />
```

Comment:
Oh right, until the `<Icon />` v1 API is merged, there is no `name` prop yet.  I know it is really close, LMK check on that.

FYI, this is one reason why Breadcrumb tests are failing on CI.